exercise-002/main.cpp

```diff
@@ -6,20 +6,50 @@
 
 auto main(int argc, char **argv) -> int
 {
+    std::srand(std::time(nullptr));
     /**
      * CLI11 is a command line parser to add command line options
-     * More inpo at https://github.com/CLIUtils/CLI11#usage
+     * More info at https://github.com/CLIUtils/CLI11#usage
+     * Test commit and push No.2
      */
     CLI::App app{PROJECT_NAME};
+    auto count = 20;
     try
     {
         app.set_version_flag("-V,--version", fmt::format("{} {}", PROJECT_VER, PROJECT_BUILD_DATE));
+        app.add_option("-c,--count", count, fmt::format("Die Variable count: {}", count));          
         app.parse(argc, argv);
     }
     catch (const CLI::ParseError &e)
     {
         return app.exit(e);
     }
+    std::vector<int> data(count);
+    fmt::print("Created a vector with {} elements\n", data.size());
+
+    for(int i = 0; i< count; i++)
+    {
+        data[i] = rand()%100;
```

Comment:
```suggestion
        data[i] = (rand()%100)+1;
```
This will generate random numbers in the range 1..100